router/Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM ghcr.io/apollographql/router:v1.32.0
+FROM ghcr.io/apollographql/router:v1.40.1
 COPY router.yaml /dist/router.yaml
 COPY rhai /dist/rhai
 
 COPY ./entrypoint.sh ./entrypoint.sh
 COPY ./wait-for-it.sh ./wait-for-it.sh
-# better safe than sorry
-RUN chmod +x ./wait-for-it.sh && \
-    chmod +x ./entrypoint.sh
```

Comment:
this was throwing an error for me, and I don't think it is needed, so I removed it 😊